wiki/Tournaments/Official_support/de.md

```diff
@@ -12,97 +10,138 @@ tags:
-  - Alternative Maßstäbe und Formate können nach alleinigem Ermessen des Turnierkomitees zugelassen werden. Die Turnierveranstalter sollten [tournaments@ppy.sh](mailto:tournaments@ppy.sh) kontaktieren, wenn sie eine Turnierunterstützung für ein nicht traditionelles Turnierformat oder eine nicht traditionelle Turniergröße beantragen.
-- Der Forumsbeitrag für das Turnier **MUSS** einen klar sichtbaren Link in einer normal großen Schriftart zum [Meldeformular für Turniere](https://tcomm.hivie.tn/reports/create) als letzten Inhalt des originalen Beitrags haben.
+  - Turniere mit einem Draft- oder Auktionsformat[^tournament-format] dürfen als Viertelfinale mit Doppeleliminierungsverfahren ausgetragen werden, sofern sie darüber hinaus die folgenden Anforderungen erfüllen:
+    - Das Turnier hat keine Beschränkung auf einen bestimmten Rangbereich.
+    - Teams bestehen aus 8 und mehr Mitgliedern.
+    - Anmerkung: Bei globalen oder auf bestimmte Kontinente beschränkte Draft- und Auktionsturnieren gilt diese Regel für bis zu 2 rangoffene Einteilungen. Für kleinere Turniere gilt diese Regel nur für eine rangoffene Einteilung.
+  - Alternative Maßstäbe und Formate können nach alleinigem Ermessen des Turnierkomitees zugelassen werden. Die Turnierveranstalter sollten im Voraus [ein Ticket erstellen](https://tcomm.hivie.tn/tickets/create) oder [tournaments@ppy.sh](mailto:tournaments@ppy.sh) kontaktieren, wenn sie eine Turnierunterstützung für ein nicht traditionelles Turnierformat oder eine nicht traditionelle Turniergröße beantragen möchten.
+- Die Forenbeiträge für das Turnier — damit sind auch Forenbeiträge für Vorveranstaltungen gemeint — **MÜSSEN** einen klar sichtbaren Link in einer normal großen Schriftart zum [Meldeformular für Turniere](https://tcomm.hivie.tn/reports/create) als letzten Inhalt des originalen Beitrags haben.
   - `https://tcomm.hivie.tn/reports/create`
-  - Dieses Meldeformular wird vom [Turnierkomitee](/wiki/People/Tournament_Committee) angesehen. Wir ermutigen alle Nutzer — Spieler wie Mitarbeiter gleichermaßen — dazu, das Formular bei Bedarf zu verwenden. Jegliche Verstöße gegen die [Erwartungen](#erwartungen), die [weiteren Prozeduren](#weitere-prozeduren) oder die [Zulassungsvoraussetzungen](#anspruchsberechtigung) sollten gemeldet werden.
-- Das Turnier erlaubt nur Spielern mit außergewöhnlichen Fähigkeiten die Teilnahme, wobei Spieler mit weniger als den folgenden Rängen im entsprechenden Spielmodus nicht zugelassen sind:
+  - Über das Meldeformular eingegangene Meldungen werden vom [Turnierkomitee](/wiki/People/Tournament_Committee) angesehen. Wir empfehlen allen Nutzern, Spieler wie Mitarbeiter gleichermaßen, das Formular bei Bedarf zu verwenden. Jegliche Verstöße gegen die [Erwartungen](#erwartungen), die [weiteren Prozesse](#weitere-prozesse) oder die [Zulassungsvoraussetzungen](#anspruch-auf-unterstützung) sollten gemeldet werden.
```

Comment:
Change `Mitarbeiter` to an alternative:

- Mitglied des Turnierteams
- Turnierpersonal

? 🤔

Reply:
"Turnierorganisatoren" perhaps?
if this is the only point left I'd suggest merging this instead

Reply:
Let's merge this for now. I will add this to my notes for later consumption.